src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -1,20 +1,31 @@
 import * as _ from 'lodash';
 
-import { sbvrUtils, hooks } from '@balena/pinejs';
+import { hooks } from '@balena/pinejs';
 
-const updateLatestRelease = async (
-	id: number,
-	{ request, api }: hooks.HookArgs,
-) => {
-	// We only track builds that are successful
-	if (request.values.status !== 'success') {
+const updateLatestRelease = async ({ request, api }: hooks.HookArgs) => {
+	if (
+		!(
+			'status' in request.values ||
+			'is_passing_tests' in request.values ||
+			'release_type' in request.values ||
+			'is_invalidated' in request.values
+		)
+	) {
 		return;
 	}
-	const release = await api.get({
+	const [release] = await api.get({
 		resource: 'release',
-		id,
 		options: {
 			$select: 'id',
+			$top: 1,
+			$orderby: { start_timestamp: 'desc' },
```

Comment:
originally i had this using `end_timestamp`, but i think `start_timestamp` is more consistent with what folks expect.

Reply:
I think it makes sense 👍

Reply:
It might be worth adding a comment about why we order by start_timestamp